### PR TITLE
Used the random_name function in vault create

### LINKED ISSUE #6381: Use the `random_name` function on the `name` argument of the `vault create` command
## Current behavior

The `name` argument of the `ockam vault create` command is calling `hex::encode()` to generate the random name in the `default_value_t` attribute.

## Desired behavior

The `name` argument of the `ockam vault create` command should be calling the [`random_name()`](https://github.com/build-trust/ockam/blob/75fa2e3ece6c8ece4a89056e5360d3b1ca19ddeb/implementations/rust/ockam/ockam_api/src/cli_state/mod.rs#L421) function.

## Implementation details

Don't forget to import the `random_name()` function with `use ockam_api::cli_state::random_name;`.

This is the argument that must be changed, where `default_value_t = random_name()`:

https://github.com/build-trust/ockam/blob/1a6aa04eb0ff79a9a2b654dddf3223c69671f7d2/implementations/rust/ockam/ockam_command/src/vault/create.rs#L22-L23

---

<i>

Some tips for newcomers:
1. Create a new branch for the issue you are working on (don't push directly to your fork's `develop` branch)
2. Run `cargo fmt`. You can configure the editor to run it automatically everytime a file is saved
3. Run `cargo clippy` to make sure the code compiles + follows rust standards/best practices
4. You don't need to keep your branch updated with `develop` until there is a merge conflict. If you still want to update it, use `git rebase develop` instead of `gir merge develop` 

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).

</i>

### PR DESCRIPTION
**Pull request by milinddethe15**
closes #6381

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/vault/create.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@ use rand::prelude::random;
 use ockam::Context;
 use ockam_api::cli_state;
 use ockam_api::cli_state::traits::StateDirTrait;
+use ockam_api::cli_state::random_name;
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/vault/create.rs:6]: Make sure to run `cargo clippy`, there is an unused import you need to remove 👍

Comment (milinddethe15) [reply implementations/rust/ockam/ockam_command/src/vault/create.rs:6]: image
I am encountering an error while executing the `cargo clippy` command and have been unable to resolve it. Could you please assist me with resolving this error?

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/vault/create.rs:6]: That's weird. Could you try running the following?

```
cargo clean
cargo build -p ockam_command
cargo clippy -p ockam_command
```

If you have any errors post them here too 👍

Comment (milinddethe15) [reply implementations/rust/ockam/ockam_command/src/vault/create.rs:6]: `cargo build -p ockam_command`, `cargo clippy -p ockam_command` were excuted successfully.
But running `cargo clippy` gave me these errors:
``` 
error: failed to run custom build command for `gio-sys v0.16.3`

Caused by:
  process didn't exit successfully: `/home/mint/ockam/target/debug/build/gio-sys-70d0d6e7ee19f88a/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-env-changed=GIO_2.0_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:warning=`PKG_CONFIG_ALLOW_SYSTEM_CFLAGS="1" "pkg-config" "--libs" "--cflags" "gio-2.0" "gio-2.0 >= 2.70"` did not exit successfully: exit status: 1
  error: could not find system library 'gio-2.0' required by the 'gio-sys' crate

  --- stderr
  Package gio-2.0 was not found in the pkg-config search path.
  Perhaps you should add the directory containing `gio-2.0.pc'
  to the PKG_CONFIG_PATH environment variable
  No package 'gio-2.0' found
  Package gio-2.0 was not found in the pkg-config search path.
  Perhaps you should add the directory containing `gio-2.0.pc'
  to the PKG_CONFIG_PATH environment variable
  No package 'gio-2.0' found

error: failed to run custom build command for `gdk-sys v0.16.0`

Caused by:
  process didn't exit successfully: `/home/mint/ockam/target/debug/build/gdk-sys-ddd31515f609d1f9/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-env-changed=GDK_3.0_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR

  --- stderr
  `PKG_CONFIG_ALLOW_SYSTEM_CFLAGS="1" "pkg-config" "--libs" "--cflags" "gdk-3.0" "gdk-3.0 >= 3.22"` did not exit successfully: exit status: 1
  error: could not find system library 'gdk-3.0' required by the 'gdk-sys' crate

  --- stderr
  Package gdk-3.0 was not found in the pkg-config search path.
  Perhaps you should add the directory containing `gdk-3.0.pc'
  to the PKG_CONFIG_PATH environment variable
  No package 'gdk-3.0' found
  Package gdk-3.0 was not found in the pkg-config search path.
  Perhaps you should add the directory containing `gdk-3.0.pc'
  to the PKG_CONFIG_PATH environment variable
  No package 'gdk-3.0' found
```

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/vault/create.rs:6]: Looks like you are missing some libraries that you need installed on your linux machine. Try running the snippet from https://tauri.app/v1/guides/getting-started/prerequisites/#setting-up-linux

```
sudo apt update
sudo apt install libwebkit2gtk-4.0-dev \
    build-essential \
    curl \
    wget \
    file \
    libssl-dev \
    libgtk-3-dev \
    libayatana-appindicator3-dev \
    librsvg2-dev
```

**Review by nazmulidris — APPROVED** [review]
@milinddethe15 Thank you for your contribution 🎉 Your code will be merged soon. We have added the `HACKTOBERFEST-ACCEPTED` label to this PR 🥳 .

**Pull request merged**